src/server/plugins/session.ts

```diff
@@ -5,12 +5,16 @@ import config from '~/src/server/config.js'
 export default {
   plugin: yar,
   options: {
+    maxCookieSize: 0, // Always use server-side storage
     cache: {
+      cache: 'session',
       expiresIn: config.sessionTimeout
```

Comment:
Are we supposed to set `expiredIn` for **forms-designer** too?

Reply:
We should do, I'll get those reviewed.